cnchi/installation/process.py

```diff
@@ -80,7 +80,7 @@ def create_metalinks_list(self):
 
         if not self.down.metalinks:
             txt = _("Cannot create download package list (metalinks). "
-                    "Check log output for details.")
+                    "Check log output in tmp/cnchi.log for details.")
```

Comment:
I am not sure what you are talking about, but I mean this should be `/tmp/cnchi.log` instead of `tmp/cnchi.log` as currently it's inconsistent.